<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2080</width>
    <height>1746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="window_camera_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>System Status</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU-0 Usage</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_gpu_usage_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU-0 Mem. Usage</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_gpu_mem_usage_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>50</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Glass Device Info.</string>
    </property>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IP :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_battery_level">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Level :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_free_storage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Free Storage :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_battery_state">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery State :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_storage_level">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Storage Level :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>32</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>62</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_battery_level_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>92</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_battery_state_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>122</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_free_storage_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>152</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_storage_level_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>182</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="table_scenario_contents">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>1461</width>
      <height>791</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>840</y>
      <width>355</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Operating Scenario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scenario_reload">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>1650</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scenario_save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save &amp;&amp; Update</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>1360</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Scenario Control</string>
    </property>
    <widget class="QPushButton" name="btn_scenario_run">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 121, 66);</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../flame-avsim-manager/ico/play_circle.png</normaloff>../flame-avsim-manager/ico/play_circle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_scenario_stop">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 121, 66);</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../flame-avsim-manager/ico/stop_circle.png</normaloff>../flame-avsim-manager/ico/stop_circle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_scenario_pause">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 121, 66);</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../flame-avsim-manager/ico/pause_circle.png</normaloff>../flame-avsim-manager/ico/pause_circle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>860</y>
      <width>561</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>co-application status</string>
    </property>
    <widget class="QTableView" name="table_coapp_status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>541</width>
       <height>231</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sound files</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_sound_files">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>30</y>
      <width>251</width>
      <height>711</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_resume">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>750</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>750</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>750</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>750</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>620</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Sound Options</string>
    </property>
    <widget class="QCheckBox" name="chk_fade_out">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply Fade Out</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_fade_in">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply Fade In</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCapture_img_png"/>
    <addaction name="actionCapture_Image_with_Keypoints_png"/>
    <addaction name="actionCapture_Image_after_time_delay_png"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuHPE_model">
     <property name="title">
      <string>HPE model</string>
     </property>
     <addaction name="actionYOLO"/>
     <addaction name="actionYOLOv2"/>
     <addaction name="actionYOLOv3"/>
     <addaction name="actionYOLOv4"/>
     <addaction name="actionYOLOv8x_pose"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuHPE_model"/>
    <addaction name="actionShow_Keypoints"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_operating_scenario"/>
    <addaction name="actionLoad_sound_files"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuCapture"/>
   <addaction name="menuProcess"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCapture_img_png">
   <property name="text">
    <string>Capture image(*.png)</string>
   </property>
  </action>
  <action name="actionCapture_Image_with_Keypoints_png">
   <property name="text">
    <string>Capture Image with Keypoints(*.png)</string>
   </property>
  </action>
  <action name="actionCapture_Image_after_time_delay_png">
   <property name="text">
    <string>Capture Image after time delay(*.png)</string>
   </property>
  </action>
  <action name="actionYOLOv8n_pose">
   <property name="text">
    <string>YOLOv8n-pose</string>
   </property>
  </action>
  <action name="actionYOLO">
   <property name="text">
    <string>YOLOv8n-pose</string>
   </property>
  </action>
  <action name="actionYOLOv2">
   <property name="text">
    <string>YOLOv8s-pose</string>
   </property>
  </action>
  <action name="actionYOLOv3">
   <property name="text">
    <string>YOLOv8m-pose</string>
   </property>
  </action>
  <action name="actionYOLOv4">
   <property name="text">
    <string>YOLOv8l-pose</string>
   </property>
  </action>
  <action name="actionYOLOv8x_pose">
   <property name="text">
    <string>YOLOv8x-pose</string>
   </property>
  </action>
  <action name="actionOpen_operating_scenario">
   <property name="text">
    <string>Open operating scenario</string>
   </property>
  </action>
  <action name="actionLoad_sound_files">
   <property name="text">
    <string>Load sound files...</string>
   </property>
  </action>
  <action name="actionShow_Keypoints">
   <property name="text">
    <string>Show Keypoints</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
